kalite/distributed/custom_context_processors.py

```diff
@@ -11,6 +11,7 @@
 
 
 def custom(request):
+    is_config_package_nalanda = True if "Nalanda" in settings.CONFIG_PACKAGE else False
```

Comment:
You can just change this to `is_config_package_nalanda = Nalanda == settings.CONFIG_PACKAGE`